power_grid_model_c/power_grid_model/include/power_grid_model/component/power_sensor.hpp

```diff
@@ -67,6 +67,8 @@ class GenericPowerSensor : public Sensor {
 };
 
 template <symmetry_tag power_sensor_symmetry_> class PowerSensor : public GenericPowerSensor {
+    static constexpr double inv_base_power = 1.0 / base_power<power_sensor_symmetry_>;
```

Comment:
So, in the end it's always fixed by some suggestions existed since last century...